routes/publish.js

```diff
@@ -0,0 +1,71 @@
+var request = require("request");
+
+function isEmpty(val) {
+  return val === null || val === undefined;
+}
+
+module.exports = function(config) {
+  return function(req, res) {
+    var project = JSON.parse(JSON.stringify(req.session.project.meta));
```

Comment:
I don't think that's necessary if we are the ones controlling what the object looks like